Makefile

```diff
@@ -1,126 +1,50 @@
-#
-# Example:
-#   make
-#   make all
-#   make all WHAT=cmd/oc GOFLAGS=-v
 all build:
-	hack/build-go.sh $(WHAT) $(GOFLAGS)
+	go build $(GOFLAGS) ./cmd/cluster-kube-scheduler-operator
 .PHONY: all build
 
-# Run core verification and all self contained tests.
-#
-# Example:
-#   make check
-check: | verify test-unit
-.PHONY: check
+verify-govet:
+	go vet $(GOFLAGS) ./...
+.PHONY: verify-govet
 
-
-# Verify code conventions are properly setup.
-#
-# Example:
-#   make verify
-verify:
-	{ \
-	hack/verify-gofmt.sh ||r=1;\
-	hack/verify-govet.sh ||r=1;\
-	hack/verify-imports.sh ||r=1;\
-	hack/verify-codegen.sh ||r=1;\
-	: TODO: reactivate: hack/verify-generated-bindata.sh ||r=1;\
-	exit $$r ;\
-	}
+verify: verify-govet
+	hack/verify-gofmt.sh
+	hack/verify-codegen.sh
+	hack/verify-generated-bindata.sh
 .PHONY: verify
 
-
-# Verify commit comments.
-#
-# Example:
-#   make verify-commits
-verify-commits:
-	hack/verify-upstream-commits.sh
-.PHONY: verify-commits
-
-# Run unit tests.
-#
-# Args:
-#   WHAT: Directory names to test.  All *_test.go files under these
-#     directories will be run.  If not specified, "everything" will be tested.
-#   TESTS: Same as WHAT.
-#   GOFLAGS: Extra flags to pass to 'go' when building.
-#   TESTFLAGS: Extra flags that should only be passed to hack/test-go.sh
-#
-# Example:
-#   make test-unit
-#   make test-unit WHAT=pkg/build TESTFLAGS=-v
-test-unit:
-	GOTEST_FLAGS="$(TESTFLAGS)" hack/test-go.sh $(WHAT) $(TESTS)
+test test-unit:
+ifndef JUNITFILE
+	go test $(GOFLAGS) -race ./...
+else
+ifeq (, $(shell which gotest2junit 2>/dev/null))
+$(error gotest2junit not found! Get it by `go get -u github.com/openshift/release/tools/gotest2junit`.)
+endif
+	go test $(GOFLAGS) -race -json ./... | gotest2junit > $(JUNITFILE)
+endif
 .PHONY: test-unit
 
-# Remove all build artifacts.
-#
-# Example:
-#   make clean
+images:
+	imagebuilder -f Dockerfile -t $(DOCKER_ORG)/origin-cluster-kube-scheduler-operator .
+.PHONY: images
+
 clean:
-	rm -rf $(OUT_DIR)
+	$(RM) ./cluster-kube-scheduler-operator
 .PHONY: clean
 
-# Build the cross compiled release binaries
-#
-# Example:
-#   make build-cross
-build-cross:
-	hack/build-cross.sh
-.PHONY: build-cross
-
-# Build RPMs only for the Linux AMD64 target
-#
-# Args:
-#
-# Example:
-#   make build-rpms
-build-rpms:
-	OS_ONLY_BUILD_PLATFORMS='linux/amd64' hack/build-rpms.sh
-.PHONY: build-rpms
-
-# Build images from the official RPMs
-# 
-# Args:
-#
-# Example:
-#   make build-images
-build-images:
-	hack/build-images.sh
-.PHONY: build-images
+, := ,
+IMAGES ?= cluster-kube-scheduler-operator
+QUOTED_IMAGES=\"$(subst $(,),\"$(,)\",$(IMAGES))\"
+
+origin-release:
+	docker pull registry.svc.ci.openshift.org/openshift/origin-release:v4.0
+	bash -c 'docker build -f <(sed "s/DOCKER_ORG/$(DOCKER_ORG)/g;s/IMAGES/$(QUOTED_IMAGES)/g" hack/lib/Dockerfile-origin-release) -t "$(DOCKER_ORG)/origin-release:latest" hack'
+	docker push $(DOCKER_ORG)/origin-release:latest
```

Comment:
looks dangerous, also by default this pusher to the official repo
I am not in favor of script silently doing `git clean`, `git commit`, `git push`, docker push` and others

Reply:
we can just print it, makes sense.

Reply:
done